### PR TITLE
Add Methods for Getting Data Sources

### PR DESCRIPTION
**Pull request by rogersnick**
- adding a few more methods for gets on data sources
- supports `/api/DataSource/` and `/api/DataSource/minimal` 
- added a type for the response of getPiiType and getPrivacyType requests for columns

### CODE REVIEW

File: src/resources/privacy/privacy.api.ts [hunk 1]
```diff
@@ -95,13 +97,13 @@ const privacyApi = (configuration: Configuration): PrivacyAPI => {
     async getPiiTypeForColumns(
       params: PiiTypeForColumnsParams,
       options?: AxiosRequestConfig
-    ): Promise<(axios: AxiosInstance, basePath: string) => AxiosPromise<string>> {
```
Comment (rogersnick) [inline src/resources/privacy/privacy.api.ts:98]: The docs say `string` but return things in this shape

**Review by rogersnick — COMMENTED** [review]
just a quick couple of get methods to help with identifying workspaces

**Review by yash-coded — APPROVED** [review]
:shipit:

**Review by AdamTranquilla — APPROVED** [review]
Looks good!

**Pull request merged**